web/src/components/storage/utils.test.js

```diff
@@ -86,3 +86,17 @@ describe("splitSize", () => {
     expect(splitSize("")).toEqual({ size: undefined, unit: undefined });
   });
 });
+
+describe("hasFS", () => {
+  it("returns true if volume has given filesystem", () => {
+    expect(hasFS({ fsType: "Btrfs" }, "Btrfs")).toBe(true);
+  });
+
+  it("returns true if volume has given filesystem regarding different case", () => {
+    expect(hasFS({ fsType: "btrfs" }, "Btrfs")).toBe(true);
+  });
+
+  it("returns false  if volume has different filesystem", () => {
```

Comment:
NP: I guess Eslint is gonna complain, but anyway

```suggestion
  it("returns false if volume has different filesystem", () => {
```